war/src/main/less/pluginSetupWizard.less

```diff
@@ -75,6 +75,12 @@
         box-sizing: border-box;
         padding: 4.2em 0 4.8em 0;
     }
+    
+    .jenkins-version {
+        float: left;
```

Comment:
Right, sorry missed this. The right area is for buttons. The version is displayed on all the dialogs.